ml4gw/waveforms/cbc/phenom_d.py

```diff
@@ -454,15 +480,18 @@ def phenom_d_phase(self, Mf, mass_1, mass_2, eta, eta2, chi1, chi2, xi):
 
         return phasing, Dphasing
 
-    def phenom_d_mrd_phase(self, Mf, eta, eta2, chi1, chi2, xi):
+    def phenom_d_mrd_phase(
+        self, Mf, eta, eta2, chi1, chi2, xi, fRD=None, fDM=None
+    ):
         alpha1 = self.alpha1Fit(eta, eta2, xi)
         alpha2 = self.alpha2Fit(eta, eta2, xi)
         alpha3 = self.alpha3Fit(eta, eta2, xi)
         alpha4 = self.alpha4Fit(eta, eta2, xi)
         alpha5 = self.alpha5Fit(eta, eta2, xi)
 
         # merger ringdown
-        fRD, fDM = self.fring_fdamp(eta, eta2, chi1, chi2)
+        if (fRD is None) or (fDM is None):
```

Comment:
For completeness, throw a RuntimeError if only one of fRD or fDM is not none.

Reply:
would be better to make the `or` into an `and`?

Reply:
Sure, but check the behavior. It's better to be explicit and to raise the error for unexpected combinations.

Reply:
yea I did that in the latest commit, could you take a look?